README.md

```diff
@@ -37,15 +33,73 @@ pip install mlx-graphs
 
 To build and install `mlx-graphs` from source start by cloning the github repo
 ```
-git clone git@github.com:TristanBilot/mlx-graphs.git && cd mlx-graphs
+git clone git@github.com:mlx-graphs/mlx-graphs.git && cd mlx-graphs
 ```
 Create a new virtual environment and install the requirements
 ```
 pip install -e .
 ```
 
-### Contributing
-#### Installing test, dev, benchmaks, docs dependencies
+## Usage
+
+### Tutorial guides
+
+We provide some notebooks to practice `mlx-graphs`.
+
+- Quickstart guide: [notebook](https://mlx-graphs.github.io/mlx-graphs/tutorials/quickstart.html)
+- Graph classification guide: [notebook](https://mlx-graphs.github.io/mlx-graphs/tutorials/graph_classification.html)
```

Comment:
I made the assumption that those URLs will work, can you confirm they are right @francescofarina?